<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Currency Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDoubleSpinBox" name="dsb_amount">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="maximum">
         <double>99999999999999.984375000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_currency">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Loading...</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pb_convert">
      <property name="text">
       <string>Convert Currency</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="lw_output"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_program">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="action_reset"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_source">
    <property name="title">
     <string>Source</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_convert">
    <property name="title">
     <string>Convert to</string>
    </property>
    <addaction name="action_select_all"/>
    <addaction name="separator"/>
    <addaction name="action_loading_placeholder"/>
   </widget>
   <addaction name="menu_program"/>
   <addaction name="menu_source"/>
   <addaction name="menu_convert"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_reset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>../../../../../../../../../../../../../../../../../../.designer/backup</normaloff>../../../../../../../../../../../../../../../../../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_select_all">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="action_loading_placeholder">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Loading...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
